apps/condo/domains/user/schema/ConfirmPhoneAction.js

```diff
@@ -11,24 +11,53 @@ const { normalizePhone } = require('@condo/domains/common/utils/phone')
 const isEmpty = require('lodash/isEmpty')
 const { sendMessage } = require('@condo/domains/notification/utils/serverSchema')
 const { ConfirmPhoneAction: ConfirmPhoneActionGQL, generateSmsCode } = require('@condo/domains/user/utils/serverSchema')
```

Comment:
it's better to use something like: `ConfirmPhoneActionUtils`
We use ConfirmPhoneActionGQL for @condo/domains/user/gql

Reply:
Done